src/utils.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  setDefaultSVG: async function(name, error) {
+    const url = `https://avatars.dicebear.com/api/initials/${name}.svg`
```

Comment:
Does this line need an await?

Reply:
Nope, its just a simple url for the image `src` attribute, it is not doing any fetch.

Reply:
`async` is not needed here so i have removed it.